app/forms/steps/shared/no_op_form.rb

```diff
@@ -4,7 +4,8 @@ class NoOpForm < Steps::BaseFormObject
       # NOTE: steps using this form do not persist anything to DB.
       # It is only used to advance in the decision tree.
       # Normally used through the `Steps::NoOpAdvanceStep` concern.
-      #
+      include Steps::ApplicantOrPartnerExclusive
```

Comment:
Not sure about this including this in this shared form. There are two dwp screens that do not have their own form as they don't persist anything (the cannot check dwp status and confirmed benefit check result). They use this form along with two other forms in our codebase (4 in total). So overall doesn't impact a lot of forms but this could increase in the future...

Reply:
I think we need to find an alternative to including this here. Happy to pair.